include/revng/PipelineC/ForwardDeclarations.h

```diff
@@ -21,3 +21,4 @@ typedef const pipeline::DiffMap rp_diff_map;
 typedef ErrorList rp_error_list;
 typedef llvm::StringMap<std::string> rp_string_map;
 typedef pipeline::InvalidationMap rp_invalidations;
+typedef llvm::SmallString<0> rp_buffer;
```

Comment:
Why `SmallString`? It's not a string in the general case.

Reply:
I can put `llvm::SmallVector<char, 0>` if it makes it clearer on the intentions

Reply:
`uint8_t` works.

Reply:
Not really, `llvm::raw_svector_ostream` wants a `SmallVectorImpl<char>` as the constructor argument